task/build-paketo-ubi-builder/0.1/README.md

```diff
@@ -0,0 +1,30 @@
+# build-paketo-ubi-builder task
```

Comment:
why it's called UBI? Is it somehow limited only to Red Hat UBI?

Reply:
Good question. It is not limited to Red Hat UBI = RHEL but as paketo community already build ubuntu images for buildpack community, then we will first focus here to build RHEL images. We can remove the `-ubi-` text if needed

Reply:
Fixed. See: 06ed6df7